agent/kaggle_cli.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
no necessary to add a new cli enrty

Reply:
An experimental CLI to improve the kaggle workflow.

Don't want to influence the baseline mode and add too many if-else before finishing this workflow.